app/src/main/java/com/example/newsapp/ui/home/fragments/EditProfileFragment.kt

```diff
@@ -44,7 +44,7 @@ class EditProfileFragment : Fragment() {
             if (result != null) {
                 filePath = result
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-                    if(!requireContext().contentResolver.equals(null)) {
+                    if (!requireContext().contentResolver.equals(null)) {
```

Comment:
@jnandrawos don't check for null this way, instead use safe calls and ?.let { }

Reply:
done